src/modules/todos.js

```diff
@@ -45,6 +76,22 @@ export const deleteToDo = (e, todos) => {
   renderToDoList(todos);
 };
 
+/* 
+const deleteToDo = (e, todos) => {
+  const deleteButton = e.target.closest('.trash-can');
+  const clickedToDo = deleteButton.previousElementSibling;
+  const toDoIndex = todos.findIndex(
+    (toDoItem) => toDoItem.toDoItem === clickedToDo.value,
+  );
+  todos.splice(toDoIndex, 1);
+  todos.forEach((toDoItem, index) => {
+    toDoItem.id = index + 1;
+  });
+  setToDo(todos);
+  renderToDoList(todos);
+}
+*/
+
```

Comment:
Please delete commented code inside your code because it makes your code unclean. 😉